src/content/docs/index.mdx

```diff
@@ -34,17 +34,17 @@ import { Card, CardGrid } from '@astrojs/starlight/components';
 		The modular build system can easily be extended by just writing a shell script.
     Modules you create can be added into the official BlueBuild repository or your own for the community to take advantage of.
 	</Card>
-	<Card title="Based on standards" icon="puzzle">
-		The format of your OS images is the familiar OCI image format, famously used by Docker and Podman.
+	<Card title="Based on solid standards" icon="puzzle">
```

Comment:
Would "stable standards" be better than "solid standards"? This phrasing stood out to me as odd, but couldn't really point out why.

Reply:
> Would "stable standards" be better than "solid standards"? This phrasing stood out to me as odd, but couldn't really point out why.

Idk, this one sounds good to me.

What do you think of this one?
`Based on well-known standards`

Oh, you don't like double-adjectives.

I guess we can include `stable` here as Containerfile standard doesn't change much or at all

Reply:
> Oh, you don't like double-adjectives.

You understood that wrong. I meant stringing two different adjectives next to each other ie. "effortless top-notch".
I actually like your suggestion.

Reply:
> > Oh, you don't like double-adjectives.
> 
> You understood that wrong. I meant stringing two different adjectives next to each other ie. "effortless top-notch". I actually like your suggestion.

Updated